README.md

```diff
@@ -110,7 +110,7 @@ Compiles the `document` AST, using an optional operationName and compiler option
     so this option should only be set to true if there are strong assurances that the values are valid.
   - `customSerializers` {Object as Map, default: {}} - Replace serializer functions for specific types. Can be used as a safer alternative
     for overly expensive serializers
-  - `customJSONSerializer` {boolean, default: false} - Whether to produce also a JSON serializer function using `fast-json-stringify`. The default stringifier function is `JSON.stringify`
+  - `customJSONSerializer` {function, default: false} - A function to be called with [`CompilationContext`](https://github.com/zalando-incubator/graphql-jit/blob/master/src/execution.ts#L87) to produce also a JSON serializer function. The default stringifier function is `JSON.stringify`
```

Comment:
```suggestion
  - `customJSONSerializer` {function, default: undefined} - A function to be called with [`CompilationContext`](https://github.com/zalando-incubator/graphql-jit/blob/master/src/execution.ts#L87) to produce also a JSON serializer function. The default stringifier function is `JSON.stringify`
```

Reply:
In code, the default value is `false`. Is this okay to write it as `undefined` in the doc (even though it does not matter actually)

https://github.com/zalando-incubator/graphql-jit/blob/8b678a10b9ec1e16db84b0566f4d64fe9be9b5de/src/execution.ts#L214